### PR TITLE
Fix kubectl 404

### LINKED ISSUE #1254: Issue with all 'See also Links' at kubectl docs
Most **kubectl** docs contain a 'See Also' section.

Links at that section should look for the content a folder up.

Example:
http://kubernetes.io/docs/user-guide/kubectl/kubectl_rollout/

links at the *See also* to 
http://kubernetes.io/docs/user-guide/kubectl/kubectl_rollout/kubectl_rollout_history.md

which doesn't exists, instead of linking to
http://kubernetes.io/docs/user-guide/kubectl/kubectl_rollout_history.md

### PR DESCRIPTION
**Pull request by janetkuo**
1. Update script to:
   - Rename kubectl.md to index.md
   - Remove "See Also" section from kubectl subcommands md files
2. Update the _date/reference.yml link to kubectl
3. Run the script to fix the 404 issue 

Fixes #1254

cc @kubernetes/docs @jaredbhatti @pwittrock @nikhiljindal

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1344)

### CODE REVIEW

**Comment by janetkuo** [comment]
Need to redirect http://kubernetes.io/docs/user-guide/kubectl/kubectl/ to http://kubernetes.io/docs/user-guide/kubectl/ also

File: update-imported-docs.sh [hunk 1]
```diff
@@ -102,6 +102,12 @@ cd docs/user-guide/kubectl
   find . -name '*.md' -type f -exec sed -i -e '/<!-- BEGIN MUNGE: IS_VERSIONED -->/,/<!-- END MUNGE: IS_VERSIONED -->/d' {} \;
   find . -name '*.md' -type f -exec sed -i -e '/<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->/,/<!-- END MUNGE: UNVERSIONED_WARNING -->/d' {} \;
 
+  # Rename kubectl.md to index.md
+  mv kubectl.md index.md
+  # Strip the "see also" links
```
Comment (nikhiljindal) [inline update-imported-docs.sh]: Add a comment saying why are we doing this?

**Comment by nikhiljindal** [comment]
Why remove the See also sections?
Seem useful to me.

**Comment by janetkuo** [comment]
Because we use relative links in the .md files, and the site treat relative links with respect to its current url, not file location. 

For example, 
In http://kubernetes.io/docs/user-guide/kubectl/kubectl_annotate/, see also links to relative link (kubectl.md), will redirect to http://kubernetes.io/docs/user-guide/kubectl/kubectl_annotate/kubectl/, whereas it links to the right file in github repo https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/kubectl/kubectl_annotate.md

File: update-imported-docs.sh (hunk 1 above)
Comment (janetkuo) [inline update-imported-docs.sh]: Done

**Comment by steveperry-53** [comment]
I don't understand why we have to remove the See Also links. We can use link targets that are not relative to the current URL, like this:
Learn more about [kubectl port-forward](/docs/user-guide/kubectl/kubectl_port-forward/).

**Comment by janetkuo** [comment]
> I don't understand why we have to remove the See Also links. We can use link targets that are not relative to the current URL, like this:
(/docs/user-guide/kubectl/kubectl_port-forward/).

This will break the link in GitHub repo. Also, since the links are auto-generated by a 3rd party tool, if we want absolute link we'll need to parse the relative links and replace them. Most "See Also" just link to kubectl anyways, except for the one in kubectl.md (which is not removed).

**Comment by nikhiljindal** [comment]
* Are we removing "See also" section because it is difficult to fix or because we do not think it is helpful.
If the latter, we should remove them from the source docs in core repo, rather than removing them while importing them to the docs repo.
* I expect most users to use the docs in k8s.io rather than the ones in github repo. So if we have to choose between the link working in only one of them, it seems to me that we should choose k8s.io.

cc @kubernetes/kubectl

**Pull request merged**